src/main.ts

```diff
@@ -0,0 +1,118 @@
+import * as core from '@actions/core'
+import * as util from 'util'
+import { ExecOptions } from '@actions/exec/lib/interfaces';
+import { ToolRunner } from '@actions/exec/lib/toolrunner';
+import { getTrivy } from './trivyHelper'
+import { getDockle } from './dockleHelper'
+const download = require('download');
```

Comment:
low priority: Keep import consistent with other imports. Same for semi-colons.

Reply:
For some modules, import statements give an error 'Could not find a declaration file for module'. That's why, I'm using require for those modules. I'll add semi-columns for all the statements.

Reply:
Consider not using this. Use this client instead: https://github.com/Azure/aks-set-context/blob/master/src/client.ts